src/igvfd/audit/file_set.py

```diff
@@ -137,27 +157,75 @@ def audit_unexpected_seqspec(value, system):
             "audit_description": "Pod5 sequence files in a file set should not be linked to a sequence specification file.",
             "audit_category": "unexpected sequence specification",
             "audit_level": "NOT_COMPLIANT"
+        },
+        {
+            "audit_description": "Sequence files should not have both `seqspecs` and `seqspec_document`.",
+            "audit_category": "unexpected sequence specification",
+            "audit_level": "NOT_COMPLIANT"
+        },
+        {
+            "audit_description": "Non-single cell sequence files are expected to link to a Document with `document_type` library structure seqspec.",
+            "audit_category": "unexpected sequence specification",
```

Comment:
I think this should be "inconsistent sequence specification document" and it should be an ERROR as well.